test_data/rules/experiments/constructor_tearoffs/rules/always_specify_types.dart

```diff
@@ -12,15 +12,30 @@ import 'package:meta/meta.dart';
 void constructorTearOffs() {
   List.filled; // LINT
   List<List>.filled; // LINT
+  // todo(pq): should be OK.
+  // List<E> Function<E>(int, E) filledList = List.filled; // OK - generic function
+  filledList<int>(3, 3); // OK
+  filledList(3, 3); // todo(pq): LINT? -- https://github.com/dart-lang/linter/issues/2914
```

Comment:
These four lines appear to have been duplicated with another PR.

Reply:
Yeah.  They're dependent.  I'll do some gitfu to merge.  Good eyes!